Hestia/Hestia/frontend/DevicesScreen/AddDeviceScreen/TableSourceAddDeviceProperties.cs

```diff
@@ -55,12 +59,35 @@ public override UITableViewCell GetCell(UITableView tableView, NSIndexPath index
 
             // Put the property names in the placeholder filed
             cell.UpdateCell(propertyNames[indexPath.Row]);
+            cell.Accessory = UITableViewCellAccessory.DetailButton;
             // Keep reference in hashtable
             inputs[propertyNames[indexPath.Row]] = cell;
 
             return cell;
 
         }
 
-    }
+		public override void AccessoryButtonTapped(UITableView tableView, NSIndexPath indexPath)
+		{
+            if(propertyNames[indexPath.Row]=="ip")
```

Comment:
This should not be hardcoded. The explanation is in the PluginInfo.RequiredInfo (which returns a Dictionary, it is the second element in the dictionary , see also the backend code).